### ISSUE TITLE
Update Data Pages

### ISSUE THREAD

**Issue by Makosak** [GeoDaCenter/covid-v3#52]
Review & Update:
https://uscovidatlas.org/docs
https://uscovidatlas.org/download

Remove? https://uscovidatlas.org/api since we flipped to download

**Comment by Makosak** [comment]
unlink API page @mukeshchugani10

**Comment by Makosak** [comment]
- @mukeshchugani10 will add correct boundaries on data doc page, and update text
- @mradamcox to review Data Download pages
- @mukeshchugani10 and @mradamcox will re-point everything to CovidV3 repo
- @Makosak will review the data doc pages for final edits/to update

**Comment by mukeshchugani10** [comment]
> unlink API page @mukeshchugani10

Implemented in #65 .

**Comment by mukeshchugani10** [comment]
> * @mukeshchugani10 will add correct boundaries on data doc page, and update text

From my discussion with @mradamcox , we found the source of these geojson files to be a S3 bucket named `geoda-covid-atlas`. This bucket contains both a geojson folder and a geojson.zip. @mradamcox will investigate which one of these is the actual source of geojson files and what the workflow `Daily Data Pull` does. For future reference, the new geojson files (based on what has been uploaded to the new mapbox account) are present [here](https://drive.google.com/file/d/1VzqbyU_i_pAd8GOnlAaVIDpxn91eGRkT/view?usp=sharing).

**Comment by mradamcox** [comment]
I've moved that updated zip file into our new S3 bucket which is described in #49. The data flow is basically:

- GH workflows pull data, process, and upload it to S3, including these zip files (this action deprecated)
- The fetch data process used during site build pulls the zip files and unzips them to the public directory.

For this ticket, I've just added a note to the metdata file describing how the highlight data is subtracted from a dissolved use geography. This note was already in there, I just made it a bit more prominent.
